nomos-services/chain/chain-service/src/lib.rs

```diff
@@ -94,6 +96,10 @@ pub enum Error {
     Storage(String),
     #[error("Blob validation failed: {0}")]
     BlobValidationFailed(#[from] blob::Error),
+    #[error("Block header not found: {0}")]
```

Comment:
nit: header/block id not found

Reply:
Fixed in https://github.com/logos-co/nomos/pull/1786/commits/aff06fa06392f49bb592cee34c8791f6f6e9f6cb